ui/perfherder/graphs/GraphsContainer.jsx

```diff
@@ -297,6 +297,8 @@ class GraphsContainer extends React.Component {
       bottom: 50,
     };
 
+    const today = new Date();
```

Comment:
This should probably be in UTC. You can use the moment library - see `this.checkDate` on line 334.